library/rethinkdb

```diff
@@ -1,16 +1,6 @@
 # maintainer: Daniel Alan Miller <dalanmiller@rethinkdb.com> (@dalanmiller)
 
-2.0.4: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.0.4
-2.0: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.0.4
-
-2.1.6: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.1.6
-2.1: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.1.6
-
-2.2.6: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.2.6
-2.2: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.2.6
-
-2.3.4: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.4
-2.3.5: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.5
-2.3: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.5
-2: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.5
-latest: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.5
+2.3.6: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.6
+2.3: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.6
+2: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.6
+latest: git://github.com/rethinkdb/rethinkdb-dockerfiles@da98484fc73485fe7780546903d01dcbcd931673 jessie/2.3.6
```

Comment:
This commit needs to be updated to `05946c0dbe3c7fa9338d3827428b2c32074a1447` (https://github.com/rethinkdb/rethinkdb-dockerfiles/commit/05946c0dbe3c7fa9338d3827428b2c32074a1447), right?

It's probably worth considering swapping to the newer file format too, since it'll make things simpler to update by hand:
```
Maintainers: Daniel Alan Miller <dalanmiller@rethinkdb.com> (@dalanmiller)
GitRepo: https://github.com/rethinkdb/rethinkdb-dockerfiles.git
GitCommit: 05946c0dbe3c7fa9338d3827428b2c32074a1447

Tags: 2.3.6, 2.3, 2, latest
Directory: jessie/2.3.6
```